src/routes/jira/connect/enterprise/app/jira-connect-enterprise-app-put.ts

```diff
@@ -58,6 +58,6 @@ export const JiraConnectEnterpriseAppPut = async (
 		});
 
 		res.status(404).send({ message: "Failed to update GitHub App." });
-		return next(new Error(`Failed to update GitHub app: ${error}`));
+		return next(new Error(`Failed to update GitHub app: ${error instanceof Error ? error.toString() : "unkown"}`));
```

Comment:
I see this construction over and over... wondering if we need some `toError(cause: unknown, message: string)` utility function?